homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -0,0 +1,87 @@
+"""Support for Aladdin Connect Garage Door sensors."""
+from __future__ import annotations
+
+from typing import cast
+
+from AIOAladdinConnect import AladdinConnectClient
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, SIGNAL_STRENGTH_DECIBELS
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .model import DoorDevice
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="battery_level",
+        name="Battery Level",
+        device_class=SensorDeviceClass.BATTERY,
+    ),
+    SensorEntityDescription(
+        key="rssi",
+        name="WIFI RSSI",
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
```

Comment:
Please disable this entity by default with the `entity_registry_enabled_default` description attribute.